CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
   - The flag `--domain-filter` can be repeated multiple times like `--domain-filter=example.com --domain-filter=company.org.`.
   - A trailing period is not required anymore for `--domain-filter` when AWS (or any other) provider is used.
-
+  - The `external-dns.alpha.kubernetes.io/hostname` annotation accepts now a comma separated list of hostnames and a trailing period is not required anymore.
```

Comment:
nitpick: this should go on the top of the changelog. The idea is once release is prepared we will just have to add the date, and what follows is in the chronological order from most recent onwards